packages/terra-clinical-item-display/src/terra-dev-site/doc/example/TextStylesDisabled.jsx

```diff
@@ -3,7 +3,6 @@ import ItemDisplay from 'terra-clinical-item-display';
 
 const component = () => (
   <React.Fragment>
-    <ItemDisplay text="Default" isDisabled />
     <ItemDisplay text="Primary" textStyle="primary" isDisabled />
```

Comment:
Lets also update this, for those looking at code examples to not think they need to add a textStyle:

```suggestion
    <ItemDisplay text="Primary" isDisabled />
```

Reply:
Updated